templates/prescription/general_edit.html

```diff
@@ -256,10 +280,32 @@
 {$ENDING_JAVASCRIPT}
 </script>
 </form>
+<script type="text/javascript" src="{$WEBROOT}/public/assets/jquery-min-1-9-1/index.js"></script>
+<script type="text/javascript" src="{$WEBROOT}/library/js/jquery-ui.min.js"></script>
```

Comment:
Why do you need jquery-ui  in this script (I couldn't find anything you added that needs this)?

Also why do you need to bring in jquery-min, which is already brought in above?